pandevice/userid.py

```diff
@@ -108,7 +108,7 @@ def batch_end(self):
             self.send(uid_message)
         self._batch_uidmessage = deepcopy(self._uidmessage)
 
-    def send(self, uidmessage):
+    def send(self, uidmessage, ips=[], tags=[]):
```

Comment:
You can't have mutable default arguments, that causes issues.  Please change the defaults to `None` for both ips and tags.  Then inside the function, set each variable to an empty list if it is `None`.  Alternatively, you can use an empty tuple as the default, since tuples aren't mutable.